x-pack/plugins/apm/public/index.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import {
+  PluginInitializer,
+  PluginInitializerContext
+} from '../../../../src/core/public';
+import { ApmPlugin, ApmPluginSetup, ApmPluginStart } from './plugin';
+
+export interface ConfigSchema {
```

Comment:
It's not entirely clear to me when reading this what this type is based on. It looks like they are the properties we want to expose from the server config to the browser, but I'm not seeing the connection to runtime code. Can we tie them together in some way?